nat-lab/tests/uniffi/libtelio_remote.py

```diff
@@ -13,6 +14,7 @@
 
 REMOTE_LOG = "remote.log"
 TCLI_LOG = "tcli.log"
+MOOSE_LOG_DIR = "moose_logs"
```

Comment:
Can this constant be used instead of other occurrences of `moose_logs`?

Reply:
I'm having a hard time forcing python to import `MOOSE_LOGS_DIR` from `moose.py` in `libtelio_remote.py` so I'm afraid this is the only place where this string has to be duplicated.

Reply:
This should be possible to solve but the current solution might be cleaner. I'm guessing your issue with importing is happening on windows and mac but kinda works on linux? In that case, in `(mac|windows)_vm_util.py` if you copy `moose.py` to a folder "next to" the `uniffi` folder in the VM you should be able to import it with a relative file import. As in, a partial file tree view on the VMs should look like
```
..
  - uniffi
    - libtelio_remote.py
  - utils
    - moose.py 
```

Though I should say that I haven't tried this so maybe it doesn't work :shrug:

Reply:
To be honest, I'm not convinced that complexity of copying that file just to avoid copying string constant is justifiable.

Reply:
Agree